### PR TITLE
feat: add WalletConnect and show disclaimer

### PR DESCRIPTION
**Pull request by Tulun**
Screen Shot 2022-03-30 at 2 07 59 PM
https://app.shortcut.com/uma-project/story/5104/re-add-walletconnect-to-across-and-if-users-are-connected-with-walletconnect-show-a-disclaimer

### CODE REVIEW

File: src/utils/onboard.ts [hunk 1]
```diff
@@ -41,9 +41,11 @@ export function OnboardEthers(config: Initialization, emit: Emit) {
       },
       wallet: (wallet: Wallet) => {
         savedWallet = wallet;
+        console.log("wallet in onboard", wallet);
```
Comment (mrice32) [inline src/utils/onboard.ts:44]: stray log?

File: src/Routes.tsx [hunk 2]
```diff
@@ -71,6 +71,15 @@ const Routes: FC<Props> = () => {
           </div>
         </SuperHeader>
       )}
+      {name && name === "WalletConnect" && (
+        <SuperHeader>
+          <div>
+            Do not change networks when submitting transfers on Across. Across
```
Comment (mrice32) [inline src/Routes.tsx:77]:
```suggestion
            Do not change networks after connecting to Across with WalletConnect. Across
```
```suggestion
            Do not change networks when submitting transfers on Across. Across
```

Comment (mrice32) [inline src/Routes.tsx:77]:
```suggestion
            Do not change networks after connecting to Across with WalletConnect. Across
```

File: src/components/SendAction/SendAction.tsx [hunk 3]
```diff
@@ -236,8 +237,18 @@ const SendAction: React.FC = () => {
         )}
 
         <PrimaryButton onClick={handleClick} disabled={buttonDisabled}>
-          {buttonMsg()}
+          <span>{buttonMsg()}</span>
         </PrimaryButton>
+        {name && name === "WalletConnect" && (
+          <WalletConnectWarning>
+            <span>
+              Do not change networks when submitting a transfer. Across is not
```
Comment (mrice32) [inline src/components/SendAction/SendAction.tsx:245]:
```suggestion
              Do not change networks after connecting to Across with WalletConnect. Across is not
```

File: src/Routes.tsx (hunk 2 above)
Comment (Tommy1231232) [reply src/Routes.tsx:77]: @Tulun let's commit this change

**Review by Gamaranto — APPROVED** [review]
agree with others comments, otherwise lgtm

File: src/utils/onboard.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (Tulun) [reply src/utils/onboard.ts:44]: Removed.

File: src/Routes.tsx [hunk 4]
```diff
@@ -71,6 +71,15 @@ const Routes: FC<Props> = () => {
           </div>
         </SuperHeader>
       )}
+      {name && name === "WalletConnect" && (
```
[the remark this answers is not in the document]
Comment (Tulun) [reply src/Routes.tsx:74]: Fair, simplified.

**Pull request merged**